priv/templates/phx.gen.auth/schema.ex

```diff
@@ -70,6 +75,19 @@ defmodule <%= inspect schema.module %> do
     end
   end
 
+  defp maybe_validate_unique_email(changeset, opts) do
+    validate_unique_email? = Keyword.get(opts, :validate_email, true)
+    email = get_change(changeset, :email)
+
+    if validate_unique_email? and email do
+      changeset
+      |> unsafe_validate_unique(:email, <%= inspect schema.repo %>)
+      |> unique_constraint(:email)
+    else
+      changeset
+    end
```

Comment:
```suggestion
    if Keyword.get(opts, :validate_email, true) do
      changeset
      |> unsafe_validate_unique(:email, <%= inspect schema.repo %>)
      |> unique_constraint(:email)
    else
      changeset
    end
```

But I think we can always add the constraint...